pvlib/iotools/pvgis.py

```diff
@@ -665,3 +665,45 @@ def read_pvgis_tmy(filename, pvgis_format=None, map_variables=None):
         data = data.rename(columns=VARIABLE_MAP)
 
     return data, months_selected, inputs, meta
+
+
+def get_pvgis_horizon(latitude, longitude, url=URL, **kwargs):
+    """Get horizon data from PVGIS.
+
+    Parameters
+    ----------
+    latitude : float
+        Latitude in degrees north
+    longitude : float
+        Longitude in degrees east
+    url: str, default: :const:`pvlib.iotools.pvgis.URL`
+        Base URL for PVGIS
+    kwargs:
+        Passed to requests.get
+
+    Returns
+    -------
+    data : pd.DataFrame
+        Pandas dataframe of the retrived horizon
+
```

Comment:
```suggestion
    data : pd.Series
        Pandas Series of the retrived horizon elevation angle. Index is the
        corresponding horizon azimuth angles.

    Notes
    -----
    The horizon azimuth is specified clockwise from north, e.g., south=180.
    This is the standard pvlib convention, although the PVGIS website specifies
    south=0.
```

Reply:
I suggest adding the above note as well as outputting a pandas Series instead of a DataFrame as there is only one colummn.